template/NuGet.Config

```diff
@@ -6,5 +6,8 @@
     <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="fsharp-daily" value="https://www.myget.org/F/fsharp-daily/api/v3/index.json" />
+    <!-- Location of dotnet-compile-fsc tool -->
+    <add key="dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/api/v3/index.json" />
+    <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
```

Comment:
This feed is not for external consumption. It is only for the dotnet/cli repo.

Whatever packages you need out of it will be on their official feed, and you should get it from there.

What package(s) do you need this feed for? If you let me know the packages, I can help you find the correct feed.